layouts/partials/site-footer.html

```diff
@@ -27,11 +27,21 @@
                 <svg class="icon gblog_heart"><use xlink:href="#gblog_heart"></use></svg>
             </span>
         </section>
-        {{ with .Site.Params.GeekblogHostedOn }}
+        {{ if or (.Site.Params.GeekblogHostedOn) (.Site.Params.GeekblogContentLicense) }}
         <section class="flex flex-wrap align-center">
+            {{ with .Site.Params.GeekblogHostedOn }}
             <span class="gblog-footer__item">
                 Hosted on <a href="{{ .link }}" class="gblog-footer__link">{{ .name }}</a>
             </span>
+            {{ end }}
+            {{ with .Site.Params.GeekblogContentLicense }}
+            <span class="gblog-footer__item">
+                {{ if site.Params.GeekblogHostedOn }}
```

Comment:
I think this block is wrong here. Why should the `GeekblogHostedOn` text be included at this position again?

Reply:
Sorry, I probably should have added a sentence to explain this: This does not include `GeekblogHostedOn` but only checks if it has been shown and prefixes `GeekblogContentLicense` with a separator if necessary. It just looked weird to have both in one line without any separator.

Reply:
No worries, :facepalm: I read this as a template `{ . }}` Ok got it. In this case, I would prefer one of the following options:

- Always add a dot to `Hosted on...` and `Content licensed under...` to make both a full sentence
- Move the License information to its own [section](https://github.com/thegeeklab/hugo-geekblog/blob/main/layouts/partials/site-footer.html#L31-L35) to always display it as a separate line

Reply:
It's actually a "middle dot", so it looks like this: "Hosted on somewhere · Content licensed under something"
I can of course also make them sentences or separate sections, though. Which do you prefer?

Reply:
Personally, I would choose option 2) if you don't mind

Reply:
I have updated the PR to use a separate section for the content license (I noticed this looks better on mobile anyway).